backend/src/routes/api/status/statusUtils.ts

```diff
@@ -61,47 +34,11 @@ export const status = async (
   }
 };
 
-export const getGroupsConfig = async (coreV1Api: CoreV1Api, namespace: string): Promise<string> => {
-  try {
-    return (await coreV1Api.readNamespacedConfigMap(GROUPS_CONFIGMAP_NAME, namespace)).body.data[
-      'groups-config'
-    ];
-  } catch (e) {
-    throw new Error(
-      `Failed to retrieve ConfigMap ${GROUPS_CONFIGMAP_NAME}, might be malformed or doesn't exist.`,
-    );
-  }
-};
-
-export const getGroupsAdminConfig = async (
-  coreV1Api: CoreV1Api,
-  namespace: string,
-  groupsConfigName: string,
-): Promise<string> => {
-  try {
-    return (await coreV1Api.readNamespacedConfigMap(groupsConfigName, namespace)).body.data[
-      'admin_groups'
-    ];
-  } catch (e) {
-    throw new Error(
-      `Failed to retrieve ConfigMap ${groupsConfigName}, might be malformed or doesn't exist.`,
-    );
-  }
-};
-
-export const getGroup = async (
-  customObjectsApi: CustomObjectsApi,
-  adminGroup: string,
-): Promise<string[]> => {
-  try {
-    const adminGroupResponse = await customObjectsApi.getClusterCustomObject(
-      'user.openshift.io',
-      'v1',
-      'groups',
-      adminGroup,
-    );
-    return (adminGroupResponse.body as groupObjResponse).users;
-  } catch (e) {
-    throw new Error(`Failed to retrieve Group ${adminGroup}, might not exist.`);
-  }
+const createCustomError = (): createError.HttpError<500> => {
```

Comment:
This is already created here -- when you rebase just pick up that effort. https://github.com/opendatahub-io/odh-dashboard/blob/main/backend/src/utils/requestUtils.ts#L3

Reply:
👌